PP-6375 Add CI command

danworth opened the pull request:
Has sub-command `compare` which compares the outcome of ci builds
between jenkins and concourse.

Filters out results prior to enabling Concourse on the 27/4/2020.

with @rorymalcolm

danworth committed:
PP-6375 Add CI command

Has sub-command `compare` which compares the outcome of ci builds
between jenkins and concourse.

Filters out results prior to enabling Concourse on the 27/4/2020.

with @rorymalcolm

rorymalcolm reviewed (dismissed):
nice work !!!

rorymalcolm commented:
actually - this doesn't work for e2e atm unless you have an access token with elevated permissions

sfount commented on pkg/cmd/root.go:
```diff
@@ -27,6 +27,7 @@ func Execute() {
 		API(),
 		Card(),
 		Toolbox(),
+    CI(),
```
Very minor but the spacing on this doesn't look happy

rorymalcolm replied:
pressed CTRL+S and fixed this!

sfount commented on pkg/ci/compareWithJenkins.go:
```diff
@@ -0,0 +1,232 @@
+func initClient() {
+	var tokenClient *http.Client
+	oauthToken := os.Getenv("PAY_CLI_GITHUB_ACCESS_TOKEN")
+	if oauthToken != "" {
```
Nice 👌

Commit:
PP-6375 Fix Spacing

danworth committed:
PP-6375 Only search end-to-end if user has access

When getting repos for `ci compare` only search private repos is the
user has "PAY_CLI_GITHUB_ACCESS_TOKEN" as in their environment.

rorymalcolm reviewed (dismissed):
👍

rorymalcolm reviewed (approved):
`gofmt`
